### DISCUSSION TITLE
How do I replace the Video Clip from the game Amanda The Adventure

### DISCUSSION THREAD

**Discussion by flickbiAv** [nesrak1/UABEA#383] [Q&A]
I came across that when replacing Raw with my video clip, when entering the game and starting the cassette, the beginning simply disappears, it was the beginning of the cassette that I replaced, I did everything right, the game sees the Video clip but for some reason it cannot play
![fdfs](https://github.com/user-attachments/assets/ead9b0be-7a82-4f2d-84ab-6823699b8cd1)

**Comment by nesrak1** [comment]
There is no plugin to replace video clips, but you can attempt to build a unity project with a video clip in it, then rename the resource file and copy the StreamedResource part of the dump from your unity project build to this game's dump.
